vng_api_common/audittrails/viewsets.py

```diff
@@ -19,6 +19,16 @@ def get_audittrail_main_object_url(self, data, main_resource):
             return data[self.audittrail_main_resource_key]
         return data[main_resource]
 
+    def get_applicatie_id(self):
+        try:
+            applicatie_id = self.request.jwt_payload['client_id']
+        except:
+            applicatie_id = self.request.META.get('X-NLX-Request-Application-Id', '')
+        return applicatie_id
+
+    def get_applicatie_weergave(self):
```

Comment:
Deze wordt alleen intern gebruikt binnen `create_audittrail` en wordt typisch niet overruled in child classes toch? Dan mag deze `_` ervoor hebben ter indicatie interne functie.

Reply:
IMO this doesn't even need to be in its own method, cluttering the viewset/mixin namespace. Should just be a couple lines in `def create_audittrail`